sci-physics/bullet/bullet-3.05.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake toolchain-funcs
+
+DESCRIPTION="Continuous Collision Detection and Physics Library"
+HOMEPAGE="http://www.bulletphysics.com/"
+SRC_URI="https://github.com/bulletphysics/bullet3/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="ZLIB"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+bullet3 doc double-precision examples extras openmp tbb test threads"
+
+REQUIRED_USE="
+	openmp? ( threads )
+	tbb? ( threads )
+"
+
+RDEPEND="
+	virtual/opengl
+	media-libs/freeglut
+	tbb? ( dev-cpp/tbb )
+"
+DEPEND="${RDEPEND}"
+BDEPEND="doc? ( app-doc/doxygen[dot] )"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.85-soversion.patch
+	"${FILESDIR}"/${PN}-2.89-no-dangling-pointer.patch
+)
+
+DOCS=( AUTHORS.txt LICENSE.txt README.md )
+
+# Building / linking of third Party library BussIK does not work out of the box
+RESTRICT="test"
+
+S="${WORKDIR}/${PN}3-${PV}"
+
+pkg_pretend() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+pkg_setup() {
+	[[ ${MERGE_TYPE} != binary ]] && use openmp && tc-check-openmp
+}
+
+src_prepare() {
+	cmake_src_prepare
+
+	# allow to generate docs
+	sed -i -e 's/GENERATE_HTMLHELP.*//g' Doxyfile || die
+}
+
+src_configure() {
+	local mycmakeargs=(
+		-DBUILD_CPU_DEMOS=OFF
+		-DBUILD_OPENGL3_DEMOS=OFF
+		-DBUILD_BULLET2_DEMOS=OFF
+		-DUSE_GRAPHICAL_BENCHMARK=OFF
+		-DINSTALL_LIBS=ON
+		-DINSTALL_EXTRA_LIBS=ON
+		-DBUILD_BULLET3=$(usex bullet3)
```

Comment:
Just a thought... since these don't pull any extra deps and they're on by default, is there a reason anyone would want to disable these? Should they just be enabled unconditionally?

I'm talking about bullet3 & double-precision.

Reply:
No specific reasons, they are just there for being consistent with previous versions.